prebuilt_swift_static_framework/prebuilt_swift_static_framework.bzl

```diff
@@ -93,17 +124,43 @@ _prebuilt_swift_static_framework = rule(
     },
 )
 
-def prebuilt_swift_static_framework(name, srcs = [], deps = []):
+def prebuilt_swift_static_framework(name, srcs = [], deps = [], **kwargs):
+    """Builds and bundles an iOS Swift static framework for third-party distribution.
+
+    This rule supports building the following:
+        $ blaze build SwiftLibrary (builds a `swift_library` that other targets can depend on)
+        $ blaze build SwiftLibraryFramework (builds an iOS Swift static framework)
```

Comment:
Should this be bazel instead of blaze :)

Reply:
Good catch! :)